hack/local-up-volcano.sh

```diff
@@ -95,9 +95,7 @@ prepare
 
 if [ "${INSTALL_MODE}" == "kind" ]; then
   kind-up-cluster
-  export KUBECONFIG="$(kind get kubeconfig ${CLUSTER_CONTEXT})"
+  export KUBECONFIG=${HOME}/.kube/config
```

Comment:
is there any reason changing this line?

Reply:
> is there any reason changing this line? l think if only the user not using `kind`, the default KUBECONFIG can be set as `${HOME}/.kube/config ` ?

Currently, if users install kind cluster and volcano with this shell script, it will be failed because it cannot get kubeconfig path from env KUBECONFIG.